modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/java/type/InfrastructureTypeMapping.java

```diff
@@ -36,6 +36,7 @@
       .put(TlsContextFactory.class, new InfrastructureType(TLS_PARAMETER_NAME, 8))
       .put(SourceTransactionalAction.class, new InfrastructureType(TRANSACTIONAL_ACTION_PARAMETER_NAME, 6))
       .put(OperationTransactionalAction.class, new InfrastructureType(TRANSACTIONAL_ACTION_PARAMETER_NAME, 7))
+      .put(TransactionType.class, new InfrastructureType(TRANSACTIONAL_TYPE_PARAMETER_NAME, 9))
```

Comment:
I don't find any serialization/deserialization test case example in the referenced commit.
Also, when I implemented `OperationTransactionalAction` and `SourceTransactionalAction` I asked Ale if I needed to something else and he told that for this cases I was not necessary.